gita/common.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+def get_config_dir() -> str:
```

Comment:
is it possible to have this funciton in `utils.py`?

Reply:
I just realized that `utils.py` imports `info.py` ...

Reply:
Yes. I originally was going to put it in `utils.py`